Messenger/AppDelegate.mm

```diff
@@ -105,6 +105,10 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
   _lastNotificationCount = @"";
 }
 
+- (void)setActiveConversationAtIndex:(NSString *)index {
+  [_webView.windowScriptObject evaluateWebScript:
+   [NSString stringWithFormat:@"document.querySelector('._1ht1:nth-of-type(%@) > a').click();", index]];
```

Comment:
I'm afraid this won't work. These CSS classes are automatically generated and might change at any point in time. Unless there are human-made identifiers, we won't be able to reliably find these DOM elements.